packages/catalog-search/src/SearchSuggestionItem.jsx

```diff
@@ -14,28 +15,44 @@ const SearchSuggestionItem = ({
       suggestionItemHandler(hit);
     }
   };
-
   return (
-    <Link to={url} key={hit.title} className="suggestion-item" onClick={handleLinkDisable}>
-      <div>
-        { /* eslint-disable-next-line react/no-danger, no-underscore-dangle */ }
-        <div dangerouslySetInnerHTML={{ __html: hit._highlightResult.title.value }} />
-        {
-          authoringOrganization && (
-            <div className="badge badge-light ml-3 font-weight-light authoring-org-badge">
-              {authoringOrganization}
+    <div>
+      {isPreQuery ? (
+        <Link to={url} key={hit.title} className="prequery-item pr-4 d-flex flex-column" onClick={handleLinkDisable}>
+          <div className="d-flex align-items-center justify-content-start">
+            <Image className="prequery-image mr-2" src={hit.card_image_url} />
+            <div className="d-flex flex-column">
+              {/* eslint-disable-next-line react/no-danger, no-underscore-dangle */}
+              <div dangerouslySetInnerHTML={{ __html: hit._highlightResult.title.value }} />
+              <div className="x-small d-flex flex-column">
+                {hit.partners[0]?.name} | {hit.learning_type}
```

Comment:
Non-blocking nit: the learning type is lowercase like 'course' but in the Figma mocks it's uppercase

Reply:
good catch! this has been updated